back/config/locales/en.yml

```diff
@@ -201,35 +201,39 @@ en:
       title_reset_your_password: 'Choose a new password'
   xlsx_export:
     column_headers:
-      input_id: 'ID'
-      latitude: 'Latitude'
-      longitude: 'Longitude'
-      location: 'Location'
-      author_fullname: 'Author name'
+      anonymous: 'Anonymous'
+      assignee_fullname: 'Assignee'
+      assignee_email: 'Assignee email'
+      attachments: 'Attachments'
       author_email: 'Author email'
+      author_fullname: 'Author name'
       author_id: 'Author ID'
-      attachments: 'Attachments'
       budget: 'Budget'
+      comments_count: 'Comments'
       cost: 'Cost'
       created_at: 'Submitted at'
-      submitted_at: 'Submitted at'
-      published_at: 'Published at'
-      comments_count: 'Comments'
-      likes_count: 'Likes'
+      description: 'Description'
       dislikes_count: 'Dislikes'
-      picks: 'Picks'
-      participants: 'Participants'
-      votes_count: 'Votes'
+      email: 'Email'
+      first_name: 'First name(s)'
+      image_url: 'Image URL'
+      input_id: 'ID'
       input_url: 'URL'
-      tags: 'Tags'
+      last_name: 'Last name'
+      latitude: 'Latitude'
+      likes_count: 'Likes'
+      location: 'Location'
+      longitude: 'Longitude' 
+      participants: 'Participants'
+      picks: 'Picks'
       project: 'Project'
+      published_at: 'Published at'
+      registration_completed_at: 'Registration completed at'
       status: 'Status'
-      assignee_fullname: 'Assignee'
-      assignee_email: 'Assignee email'
-      image_url: 'Image URL'
+      submitted_at: 'Submitted at'
+      tags: 'Tags'
       title: 'Title'
-      description: 'Description'
-    anonymous: 'Anonymous'
+      votes_count: 'Votes'
```

Comment:
Reordered alphabetically + added `first_name`, `last_name`, `email` & `registration_completed_at`.

This introduces a little duplication of translations, but since they are very simple, and there is no real rush, I thought the improvement in organisation by having all XLSX header translations in one place is more valuable.

Reply:
Which duplication do you mean? Between copy for XLSX and copy that is used outside XLSX? In that case I would say it's acceptable. But if there are two XLSX sheets using e.g. "first name" with different translation keys, then I think it would be an improvement to use the same translation key for all occurrences.